CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.7.0 (2019-06-26)
+
+- Updated the `deploy` command to always record the explicit version and not update the major version for pre-releases.  [#47](https://github.com/blackbaud/skyux-deploy/pull/47)
```

Comment:
Does meaning change if you say "and maintain" instead of "and not update"? So

Updated the `deploy` command to always record the explicit version and maintain the major version for pre-releases.

Reply:
I like that, would you be okay with "but" instead of "and?"

```
Updated the deploy command to always record the explicit version but maintain the major version for pre-releases.
```

Reply:
Works for me :heavy_check_mark:

Reply:
Updated.  Thanks Denise.